internal/services/containers/kubernetes_flux_configuration_resource_test.go

```diff
@@ -608,7 +671,7 @@ resource "azuread_application" "test" {
 }
 
 resource "azuread_service_principal" "test" {
-  application_id = azuread_application.test.application_id
+  client_id = azuread_application.test.client_id
 }
 
 resource "azuread_service_principal_password" "test" {
```

Comment:
I believe this should be as below?

```suggestion
resource "azuread_service_principal_password" "test" {
  service_principal_id = azuread_service_principal.test.id
}
```